pkg/kubelet/server/server.go

```diff
@@ -220,6 +220,7 @@ func ListenAndServePodResources(socket string, podsProvider podresources.PodsPro
 		os.Exit(1)
 	}
 
+	klog.InfoS("Starting to serve the podresources API", "socket", socket)
 	if err := server.Serve(l); err != nil {
```

Comment:
Is this creating a Unix Domain socket or named pipe?

Reply:
both: it's a unix domain socket on unix[1] and a named pipe on windows. Maybe should I rename the to read
```
klog.InfoS("Starting to serve the podresources API", "endpoint", endpoint)
```
(of course rename the variable not just the log)

[1]: ```
root@controlplane:/home/vagrant# file /var/lib/kubelet/pod-resources/kubelet.sock 
/var/lib/kubelet/pod-resources/kubelet.sock: socket
```

Reply:
Oh I see now, `pkg/kubelet/util/util_windows.go` has the logic to create the namepipe server.  Thanks!